datasets/ziekte_plagen_exoten_groen/dataset.json

```
@@ -0,0 +1,303 @@
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "nietBereikbaarMetHoogwerker": {
+            "description": "Niet bereikbaar voor hoogwerker",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "nietToegankelijkVoorBestrijding": {
+            "description": "Niet toegankelijk voor bestrijding",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "notitieToezichthouder": {
+            "description": "Notitie van de toezichthouder",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "omschrijvingMelding": {
+            "description": "Omschrijving van een melding",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "omschrijvingAlgemeen": {
+            "description": "Algemene omschrijving",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "opmerkingUitvoerendeOrganisatie": {
+            "description": "Opmerkingen uitvoerende organisatie",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "startInspectiedatum": {
+            "description": "Start inspectiedatum",
+            "type": "string",
+            "format": "date",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "uiterlijkeInspectiedatum": {
+            "description": "De uiterlijke inspectiedatum",
+            "type": "string",
+            "format": "date",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "uitvoerendeOrganisatie": {
+            "description": "Naam uitvoerende organisatie",
+            "type": "string",
+            "auth": "FP/MDW",
+            "reasonsNonPublic": "5.2 3: Zwaarwegend belang: persoonlijke beleidsopvattingen (bevat milieu-informatie)"
+          },
+          "validTill": {
```

Volgens mij hebben we geen regel over Engels vs. Nederlands, maar dit is het enige veld met een Engelse naam.

```suggestion
          "geldigTot": {
```

?

---

Goed punt, het attribuut kwam zo uit de gisib api, eens dat het netter staat in het nederlands.